web-app/src/main/java/com/google/sps/servlets/ReportServlet.java

```diff
@@ -43,17 +43,10 @@
 @WebServlet("/report")
 public class ReportServlet extends HttpServlet {
 
-  private static Map<String, String> PARAM_DEFAULT_MAP = new HashMap<String, String>() {
-    {
-      put("title", "");
-      put("latitude", "0.0");
-      put("longitude", "0.0");
-      put("timestamp", "0");
-      put("incidentType", "etc");
-      put("description", "");
-    }
-  };
+  private static List<String> COMPULSORY_PARAM_KEY = Arrays.asList("title", "latitude", "longitude", "timestamp",
+      "incidentType", "description");
```

Comment:
I was thinking of making it compulsory since one of the goals in our design doc was to provide context! Without descriptions, there wouldn't be a lot of context.